substrate/frame/authorship/src/lib.rs

```diff
@@ -36,10 +35,11 @@ pub trait EventHandler<Author, BlockNumber> {
 
 #[frame_support::pallet]
 pub mod pallet {
-	use super::*;
 	use frame_support::pallet_prelude::*;
 	use frame_system::pallet_prelude::*;
 
+	use super::*;
+
```

Comment:
?

Reply:
This has just been moved down, as pointed here https://github.com/paritytech/polkadot-sdk/pull/1790#discussion_r1345831064 this reformatting needs to be undone

Reply:
All fixed. Thx for noticing.